integration/tests/basic/ttt.json

```diff
@@ -0,0 +1,261 @@
+{
+    "jsonrpc": "2.0",
```

Comment:
I think you committed this file accidentally

Reply:
Removed ✅